tests/shared/utils/schemas/test_events.py

```diff
@@ -27,4 +89,16 @@ def test_remote_action_validate_all_event_subclasses(event_class: Type[Event]):
         "wrong_action",
         "warning_predicted",
     ]:
-        jsonschema.validate(response, RemoteAction.action_response_format_spec())
+        validate(response, RemoteAction.action_response_format_spec())
+
+
+def test_validate_single_event():
```

Comment:
Doesn't this [existing test](https://github.com/RasaHQ/rasa/blob/d7456ee04bd55d49468ee82a70a123a73f9fe6f8/tests/core/test_actions.py#L507) already check for valid events?